pkgs/applications/misc/get_iplayer/default.nix

```diff
@@ -11,8 +11,8 @@ perlPackages.buildPerlPackage rec {
     sha256 = "+ChCF27nmPKbqaZVxsZ6TlbzSdEz6RfMs87NE8xaSRw=";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ perl ] ++ lib.optional stdenv.isDarwin shortenPerlShebang;
+  nativeBuildInputs = [ makeWrapper ] ++ lib.optional stdenv.isDarwin shortenPerlShebang;
+  buildInputs = [ perl ];
```

Comment:
This could be a seperate commit with subject: "get_iplayer: Fix: Switch buildInputs and nativeBuildInputs"